data/gala.gschema.xml

```diff
@@ -69,6 +70,11 @@
         followed by a ':' and then the command. ';;' is used as delimiter between command definitions.
       </description>
     </key>
+    <key type="b" name="disable-hotcorners-in-fullscreen">
```

Comment:
Can we make this enabled instead of disabled? I think for the setting in the UI we probably want to bind to a switch and it would be kind of confusing logic to turn a switch on to turn the feature off, so we'd want to do an inverse bind 😅